cypress/e2e/awx/resources/inventoriesConstructed.cy.ts

```diff
@@ -158,14 +150,55 @@ describe('Constructed Inventories CRUD Tests', () => {
       });
   });
 
-  it.skip('shows a failed sync on the constructed inventory if the user sets strict to true and enters bad variables', () => {
-    //Create a constructed inventory in the beforeEach hook
+  it('shows a failed sync on the constructed inventory if the user sets strict to true and enters bad variables', () => {
```

Comment:
This feel like its testing the API more than its testing the UI.  In essence I think what we should care about is that we can submit a new inventory, I think other tests have covered that, and that we show the sync status on the details page..... that could be component test.  

I this test is solid and hasn't show any flake over several runs I say keep it.  Otherwise, I think we can remove it.

Reply:
Okay! I agree that it'd be okay to remove if it's flaky, but it so far seems to be passing over several local runs both downstream and upstream. I can also run the PR check here again to ensure that it isn't flaky, just to be sure!